<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOverWindow</class>
 <widget class="QMainWindow" name="GameOverWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_GameOver">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>-20</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Game Over!</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="ScoreFinal_Red">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="ScoreFinal_Blue">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_main_menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>491</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_WhoWon">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;We're all winners if we have fun :)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>272</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>Background</zorder>
   <zorder>Label_GameOver</zorder>
   <zorder>ScoreFinal_Red</zorder>
   <zorder>ScoreFinal_Blue</zorder>
   <zorder>button_main_menu</zorder>
   <zorder>Label_WhoWon</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
